src/sagemaker/exceptions.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
```

Comment:
the (also recently added) [pylint check](https://devbuildsagemakerpythonsdkstackpdx-sage-buildlogs-1vw05gzut3ch7.s3.amazonaws.com/8f3c333d-8cd9-427c-bda1-0e34186e8972/build.log?AWSAccessKeyId=ASIASCMSNBKC2Y2WT27Y&Signature=a0gjPkOf02f9cQQGRXNjma69srA%3D&x-amz-security-token=AgoJb3JpZ2luX2VjEHAaCXVzLXdlc3QtMiJGMEQCIGjauMPQjjtXT%2Bg%2B2ntn4N4aUOuj5FmembNvsb3%2BtVgxAiBPbnAFhvpXErtqEhUY%2FgAoJNmPF3RTdGc7dKHrYW8WXirEAgjZ%2F%2F%2F%2F%2F%2F%2F%2F%2F%2F8BEAEaDDE0MjU3NzgzMDUzMyIMHp7jVo%2FGYeGUYbOAKpgCb8c%2FDhrskrDQKhHt4NSHzjw%2BEDZ8br2%2BuO9qH4%2FP1bkTj28vimQjIRhwjYRsDpIff95eocmW2548WcJC7cByojbQR0AHgl2Hh%2Fv%2ByGBn3gWra%2Fglsm1VTklSiS3RRf%2F3yHJded4Fx%2F1ieoiNbHWnIjxLornYqnSt76o%2BLsdDxdlyrQyMGd0vdDmekt3b%2Frx9aindJPei106pkzFHIsSlCi1Cx4t3P9m7u0%2BcH0O5mtHwhaSJ7Gt6vNwo3xBsfOKKimG0Zxg6XLVAb0ZIJPnhT616hYSH6xMytif2Yqh6z0m4QuidIcbeLUZ3DbkYinV7uQLrYfZKpu6xQMztmFgyggeIWg5ossVY7jfqZlnBn%2F%2FotNCi79RwfTCm4NzpBTq1AcbV1ZqZIkf3DZ7YxpFbe%2BqEUFu3czfQQi2PF2mSZBPFYh0dYq272n4ssVM%2ByZWjMlkN6n%2Bsu6smlcMckZX2Sy2pwEv2qOocNrO5A7cVllEEb1hCYIdrhIeQVK9yFsfgMHBmNhs5wUcO38EYM1wvnFtfH%2FoUceYy97udmyG%2FEI995dsbn0%2BU%2BjJhwoUKMg%2BeGAAJ4McbILBUXNZD%2BLpCk5Xh2Ve8UsJCRX2PF%2FvtAEjU9u8HKkg%3D&Expires=1563898496) is now asking for a docstring about this module. could you add one? sorry about the back and forth!

Reply:
@laurenyu 
Strict checks are good :)
Fixed